docs/security/index.md

```diff
@@ -0,0 +1,65 @@
+# Security Hardening
+
+sda-spring-boot-commons changes some default configuration for security reasons.
+This document provides a brief overview about the addressed risks.
+
+## Risk: Accessing critical resources from untrusted environments
+
+To avoid exposing internal resources, Spring Boot Actuator is configured to listen on a separate
+port.
+Health, metrics and other sensitive information can't be exposed to the internet by accident, e.g.
+by missing to exclude the actuator path.
+
+Custom critical resources can be exposed at the management port by implementing
+`org.springframework.boot.actuate.endpoint.web.annotation.RestControllerEndpoint` or
+`org.springframework.boot.actuate.endpoint.web.annotation.ControllerEndpoint`.
```

Comment:
I'm not an expert here but it seems like this approach is discouraged:
See https://github.com/spring-projects/spring-boot/issues/31768

Seems like they prefer `@ReadOperation`, `@WriteOperation`, ...

Reply:
Documentation updated.